src/main/java/at/ac/tuwien/sese/g09/domain/Room.java

```diff
@@ -39,7 +48,8 @@ public class Room implements Serializable {
     private Set<RoomPrice> prices = new HashSet<>();
 
     @OneToMany(mappedBy = "room")
-    private Set<RoomPicture> roomPictures;
+    @JsonIgnoreProperties(value = { "room" }, allowSetters = true)
```

Comment:
To avoid an unckecked recursion

Room.roomPicture -> RoomPicture.room -> Room.roomPicutre -> RoomPicture.room > ...
We just remove the room from the roomPicture when converting to JSON